target/scripts/startup/setup-stack.sh

```diff
@@ -399,7 +399,7 @@ function _setup_postgrey
   _log 'debug' 'Configuring Postgrey'
 
   sed -i -E \
-    's|, reject_rbl_client zen.spamhaus.org$|, reject_rbl_client zen.spamhaus.org, check_policy_service inet:127.0.0.1:10023|' \
+    's|, reject_rbl_client zen.spamhaus.org=127.0.0.[2..11]$|, reject_rbl_client zen.spamhaus.org=127.0.0.[2..11], check_policy_service inet:127.0.0.1:10023|' \
```

Comment:
Isn't `[2..11]` interpreted as character class and should be escaped?

Reply:
Probably, yes - thanks :)